### DISCUSSION TITLE
Communicate the Life cycle of Tutorials to learners

### DISCUSSION THREAD

**Discussion by avallecam** [epiverse-trace/epiverse-trace.github.io#139] [General]
Lessons in the [tutorials](https://github.com/epiverse-trace/tutorials/) repo are following an iterative development life cycle ([ref](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development)). Currently in pre-alpha stage. 

This stage of development is labelled into three sections:
- **Topic tags** in the About section on GitHub ([location](https://github.com/epiverse-trace/tutorials/)) (following [workbench template guidelines](https://github.com/carpentries/workbench-template-rmd#configure-a-new-lesson))
- **Lifecycle badges** in README: lifecycle experimental and repo status WIP ([location](https://github.com/epiverse-trace/tutorials#tutorials)) (following [dev stages of open source software](https://epiverse-trace.github.io/posts/comm-software-devel/#lifecycle-badges))
- **Lesson lifecycle** in the upper left corner of the URL lesson ([location](https://epiverse-trace.github.io/tutorials/)) (defined in the [`config.yaml` of the repo](https://github.com/epiverse-trace/tutorials/blob/main/config.yaml#L24))

These could be visible to competent practitioners in open software development, but probably not as much for [our learner personas](https://epiverse-trace.github.io/tutorials/profiles.html).

We can complement them with more visual cues:
- **One callout in the Front page Summary section:** We can add one callout box (see [list of variations](https://carpentries.github.io/sandpaper-docs/instructor/component-guide.html#callout-blocks)) explaining briefly the current dev-stage of the tutorials, after the `Summary` and before the `Software Setup` of the current front page.
- **Share about the Communication channels for these tutorials:** As part of the design features for this repo, we want to guide learners to communicate via the Discussions tab. We can add this to the same callout box above with links to the Support and Contributing files as more visible reminders on the front page. (For an example of this, [refer to a previous tutorial](https://epiverse-trace.github.io/research-compendium/beforestart.html#roll-call))

Happy to read your thoughts and alternatives to this.

**Comment by Bisaloo** [comment]
Relevant: I've pulled the latest changes from `carpentries/varnish@main` and the lesson lifecycle icon is now much more prominent: 

![image](https://github.com/epiverse-trace/epiverse-trace.github.io/assets/10783929/bc5810ee-1350-4a6a-b3a8-7f9307843dda)

**Reply by avallecam** [reply]
Thank you for that update! That is a good emphasis.

Should we update the link? It currently redirects to the [CDH](https://cdh.carpentries.org/the-lesson-life-cycle.html#early-development-pre-alpha-through-alpha), which is explicit and informative for lesson developers (the intended audience of the CDH), but not as direct in the message as [the section on iterative development](https://carpentries.github.io/lesson-development-training/lesson-design.html#iterative-development).

**Reply by avallecam** [reply]
I think we can wait for an update in their side as notified in this https://github.com/carpentries/varnish/issues/41

**Comment by avallecam** [comment]
closing. Stage now visible in from learn page https://epiverse-trace.github.io/learn.html and with an appropriate visibility on tutorial heading
